samples/sm_host_shell/boards/nrf54l15dk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
+		ncs,sm-uart = &uart30;
+	};
+};
+
+/* Shell UART */
+&uart20 {
+	current-speed = <115200>;
+	status = "okay";
+};
+
+/* Serial Modem Host Shell <-> Serial Modem UART */
+/* Note: Disable VCOM0 from Board Configurator to release UART30 for SM. */
+&uart30 {
+	compatible = "nordic,nrf-uarte";
+	current-speed = <115200>;
+	status = "okay";
+	hw-flow-control;
+};
+
+/* DTR gpios for uart30 */
+/ {
+	dte_dtr: dte_dtr {
+		compatible = "nordic,dte-dtr";
+		dtr-gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
+		ri-gpios = <&gpio1 12 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>;
```

Comment:
All other UART related pins are `GPIO_ACTIVE_LOW | GPIO_PULL_UP`.

Why do we have the reverse polarity here?